### ISSUE TITLE
Missing a provider to automatically add country number in phone

### ISSUE THREAD

**Issue by Roshyo** [Setono/SyliusKlaviyoPlugin#75]
Right now, if your shop does not force the country code in phone number (like 0102030405 instead of +33102030405) then pushing to Klaviyo fails.

It should probably ignore the phone if missing that. Or even better, guess the country code according to the default address country (if existing)

**Comment by loevgaard** [comment]
It could use https://github.com/brick/phonenumber to guess it

**Comment by Roshyo** [comment]
I don't think so, because that lib assumes you know it. It can't apparently guess from just the numbers.
My problem is that in France, it's very common to give number like `0102030405` and that's it.
So then sending to Klaviyo fails since it expects the +33 instead of 0
